src/utils/SiriusAdapter.cpp

```diff
@@ -421,8 +421,15 @@ class TOPPSiriusAdapter :
       sirius_workspace_directory = String(sw_dir.absolutePath());
       
       // move tmp folder to new location
-      File::copyDirRecursively(tmp_dir, sirius_workspace_directory.toQString());
-      LOG_WARN << "Sirius Workspace was moved to " << sirius_workspace_directory << std::endl;
+      bool copy_status = File::copyDirRecursively(tmp_dir, sirius_workspace_directory.toQString());
+      if (copy_status)
+      { 
+        LOG_WARN << "Sirius Workspace was successfully moved to " << sirius_workspace_directory << std::endl;
```

Comment:
maybe LOG_INFO is better than LOG_WARN

Reply:
If you want to move, why copy?

Reply:
That is a good point.
I thought that it may be a good idea to copy and still have a temporary Version if debug >= 2.
If that is not necessary I probably can use the rename function of QT to move the directory.

Will leave the copyDirRecursively in File class - if that is ok.